README.md

```diff
@@ -140,9 +140,7 @@ Note that Ray Client uses the [ray://](https://docs.ray.io/en/master/cluster/ray
 
 ## Running tasks on a Ray remote cluster
 
-When using the `RayTaskRunner` with a remote Ray cluster, you may run into issues that are not seen when using a local Ray instance. To resolve these issues, we recommend taking the following steps when working with a remote Ray cluster:
-
-1. By default, Prefect will not persist any data to the filesystem of the remote ray worker. However, if you want to take advantage of Prefect's caching ability, you will need to configure a remote result storage to persist results across task runs. 
+When using the `RayTaskRunner` with a remote Ray cluster, you may run into issues that are not seen when using a local Ray instance. To resolve these issues, it is necessary to configure prefect remote result storage.
```

Comment:
I think we can leave point 1 to not disturb the list and make the wording more direct. Let me know how this wording reads for you. 

```suggestion
When using the `RayTaskRunner` with a remote Ray cluster, you may encounter issues not seen when using a local Ray instance. To resolve these issues, we strongly recommend taking the following steps when working with a remote Ray cluster:

1. Configure remote result storage to [persist results](https://docs.prefect.io/latest/concepts/results/?h=results#persisting-results) across task runs.
```